src/components/GrampsjsOidcButton.js

```diff
@@ -0,0 +1,323 @@
+import {html, css, LitElement} from 'lit'
+
+/**
+ * Styled OIDC provider login button component.
+ *
+ * Follows official branding guidelines for Google, Microsoft, GitHub and other providers.
+ * Includes proper logos, colors, and accessibility features.
+ */
+class GrampsjsOidcButton extends LitElement {
```

Comment:
Can you please use `<md-` elements for the buttons and `grampsjs-icon` for the icons, for consistency, please?